platform/api/_partials/resources/runners/reference.mdx

```diff
@@ -2224,7 +2224,7 @@ for each node that meets all of the scheduling requirements (resource
 request, requiredDuringScheduling anti-affinity expressions, etc.),
 compute a sum by iterating through the elements of this field and adding
 "weight" to the sum if the node has pods which matches the corresponding podAffinityTerm; the
-nodes with the highest sum are the most preferred.
+node(s) with the highest sum are the most preferred.
```

Comment:
Could you please change here and in all other occurrences of noun(s) to just plural form?